packages/suite/src/components/suite/NavigationBar/components/MainNavigation/index.tsx

```diff
@@ -68,6 +68,32 @@ const ItemTitle = styled.span<ComponentProps>`
     color: ${props => props.theme.TYPE_LIGHT_GREY};
     font-weight: ${variables.FONT_WEIGHT.MEDIUM};
 
+    &:after {
+        content: '';
+        position: absolute;
+        top: -4px;
+        left: -4px;
+        right: -4px;
+        bottom: -4px;
+        z-index: -1;
+        border-radius: ${variables.BORDER_RADIUS.HOVER};
+        transition: all ${variables.HOVER_TRANSITION.DURATION}
+            ${variables.HOVER_TRANSITION.ANIMATION};
+        background-color: transparent;
+    }
+
+    &:hover,
+    &:focus,
+    &:active {
+        &:after {
+            background-color: ${props => props.theme.BG_HOVER_ITEM};
+            top: -8px;
+            left: -8px;
+            bottom: -8px;
+            right: -8px;
+        }
+    }
+
```

Comment:
![Screenshot 2021-06-21 at 12 10 44](https://user-images.githubusercontent.com/33235762/122745843-b725ea80-d289-11eb-85ca-e7e2a53ac46e.png)

The size of padding in Figma is much bigger.

Reply:
94822a6